tests/test_transform.py

```diff
@@ -39,3 +45,24 @@ def test_pairwiseSeq_2():
     test_list = [1, 2, 3, 4]
     result = distill.pairwiseSeq(test_list, split=True)
     assert result == ((1, 2, 3), (2, 3, 4))
+    
+data = testing_utils.setup(os.path.join(DATA_DIR, "sample_data.json"), "integer")
+logs = data[1]
+
+def input_rule(log):
+    return "target" in log and "input" in log["target"]
+
+def test_label_features_1():
+    result = distill.label_features(logs,[FeatureDefinition(input_rule, "input_target")])
+    assert result
+
+def test_label_features_2():
+    with pytest.raises(TypeError):
+        result = distill.label_features(logs,[FeatureDefinition(input_rule, 10)])
+
+def test_label_features_2():
+    with pytest.raises(TypeError):
+        result = distill.label_features(logs,[FeatureDefinition("input_rule", "input_target")])
```

Comment:
These tests are fine as they test expected behavior. Two things:

1. These two tests are testing the `FeatureDefinition` class, not label_features function. So we should remove the calls to label_feature. Instead, all you need to do is try to instantiate the FeatureDefinition class with the bad arguments.
2. When creating tests, best practice is to use informative names for the test functions. These should tell the use what behavior is behaving tests, e.g.:
`def test_feature_definition_does_not_accept_non_string_label"
`def test_feature_definition_does_not_accept_non_callable_rules`